### ISSUE TITLE
some LightQuickFixTestCase tests fail due to fileEditors being null after updating plugin

### ISSUE THREAD

**Issue by SingingBush** [JetBrains/gradle-intellij-plugin#741]
Have there been and changes that would affect _LightQuickFixTestCase_? The reason I ask is that since updating to recent version of this plugin it seems that some tests have become flaky. 

```
    CompositeException (2 nested):
    ------------------------------
    [0]: java.lang.IllegalArgumentException: Argument for @NotNull parameter 'fileEditors' of com/intellij/codeInsight/daemon/DaemonCodeAnalyzer$DaemonListener.daemonFinished must not be null
    [1]: java.lang.IllegalArgumentException: Argument for @NotNull parameter 'fileEditors' of com/intellij/execution/lineMarker/RunnableStatusListener.daemonFinished must not be null
```

Perhaps this is a coincidence but I had not seen this problem prior to updating gradle-intellij-plugin

**Comment by YannCebron** [comment]
are you 100% sure nothing else changed? e.g. on CI or test environment? please post full stacktrace

**Comment by SingingBush** [comment]
There wasn't anything useful in the log prior to that. 

I'm 100% sure that without changes the tests sometimes all pass and can potentially fail. I just re-ran the tests https://github.com/intellij-dlanguage/intellij-dlanguage/actions/runs/1028527569 and this time they all passed. Beforehand the tests on Intellij 2021.1.1 had failed. No commits in between, just re-running the job. Annoying on github actions re-running means that I cannot show the result from before. Also previously to that the same test passed on 2021.1.1 and instead failed on 2021.1.3.

Certainly some horrible race condition. Hopefully it won't show up too often.

**Comment by YannCebron** [comment]
Reference https://github.com/intellij-dlanguage/intellij-dlanguage/runs/3061842318?check_suite_focus=true#step:4:1990

this looks like https://youtrack.jetbrains.com/issue/IDEA-267214

out of curiosity, why do you build/run tests against every single bugfix release? It should be pretty safe to assume users will use latest available bugfix version in general.

**Comment by SingingBush** [comment]
I don't always but wanted to see stable builds with all 2021.1.* prior to removing some

**Comment by SingingBush** [comment]
I don't have access to that youtrack ticket (am logged in). Thanks for looking in to it

**Comment by YannCebron** [comment]
Sorry, issue should be visible now (fixed in 2021.2)
